test/integration/integration_test.go

```diff
@@ -69,6 +70,10 @@ func (suite *HealthSuit) SetupTest() {
 	gopath := os.Getenv("GOPATH")
 	configFilePath := gopath + "/src/github.com/simelo/rextporter/test/integration/simple.toml"
 	suite.srv = exporter.ExportMetrics(configFilePath, 8081)
+	for i := 0; i < 3; i++ {
+		t := time.NewTimer(time.Second)
+		<-t.C
+	}
```

Comment:
@simelo what do you think about this hack? I want to wait for the server to be started before start running the tests
and I prefer to wait three times for a second than a single time for three seconds.